cmd/launcher/doctor.go

```diff
@@ -13,6 +13,9 @@ import (
 )
 
 func runDoctor(args []string) error {
+	attachConsole()
```

Comment:
should we log any returned errors?

Reply:
If `attachConsole` fails, nothing will be printed without the user piping to `Out-String`, so I wasn't sure if printing the error message would be very useful. And since this is end user-facing, I wasn't sure if printing error messages about this failure would be appropriate, either. Thoughts?

cc @directionless since you left a similar comment.

Reply:
Log them? I mean, sure, probably no one is going to look. But if someone ever files a bug report they'll be there...

Reply:
The logs for doctor only ever get written to stdout (unless someone happens to specify the root dir argument when running doctor), so they won't be available in the log file. They'll only be printed if the user is piping to `Out-String`, hence why I'm not sure if it's useful or appropriate.